jpo-ode-svcs/src/main/resources/application.yaml

```diff
@@ -149,9 +154,10 @@ management:
     web:
       exposure:
         include: prometheus
-  metrics:
-    export:
-      prometheus:
+  prometheus:
```

Comment:
@drewjj my IDE was saying the old way of enabling Prometheus was deprecated and that this was the new supported way of doing it.